src/collections/_documentation/workflow/releases/health.md

```diff
@@ -0,0 +1,137 @@
+Data filters are only partially supported for session events, and they might be working slightly different:
+
+### Equivalent Filters
+
+**release filter**
+This filter discards events of a particular release. If sessions are started or updated and would match this filter, sessions are silently discarded entirely. This is consistent with error events.
+
+**web crawler filter**
+When session events send their user agent, they are discarded the same way as error events are. There might be a slight difference in the fallback case where the user agent of the sending SDK is used (browser JavaScript).
+
+**legacy browsers**
+This works exactly like the web crawler filter. No sessions are created or updated for legacy browsers when enabled.
+
+**ip filter**
+Throws away sessions of a specific IP.
+
+#### Unsupported Filters
+
+**error message filter**
+This filter does not apply for sessions. Even if a user is discarding such errors, they will count against their stability score. An alternative measure *could be* to discard such sessions entirely if an error of this type is filtered out, but we believe this would make the system harder to understand.
+
+**browser extensions**
+Since the logic for detecting such errors requires all the error information, we cannot support this filter at the session ingest time. If we want to improve this case, we could retroactively discard such sessions entirely once we identify such an error case.
+
+## Release Health Glossary
+
+#### Session
+
+The time when the user is interacting with the application. (The application is in foreground)
+
+- Session begins with the start of the application. Or, it begins with bringing the already started application back from background to the foreground.
+
+- Session ends with the closing of the application or with the application being sent to the background.
+
+- If the application is in the background for less than 30 seconds, we do not need to start the session again.
+
+- Applications that are active even on the background (for example, music player) should track the sessions separately for the background process.
+
+#### Active Users
+
+Number of users that started the application at least once in the specified time period.
+
+#### Crash
+
+The fatal error that caused the crash of the application. Errors that did not cause the end of the application should not be included.
+
+#### Crash Free User
+
+Percentage of the users that did not experience a crash during the specified time period.
+
+#### Crash Free Sessions
+
+Number of sessions in the specified time range that did not end by the crash of the application.
+
+#### Crashed Users
+
+Number of users that experienced a crash in the specified time range.
+
+#### Release Adoption
+
+Number of users that started at least once during the specified time period of a specific release
```

Comment:
I believe this only would work if release health is enabled for everyone.
if a customer decides to do a A/B test with the release health feature, the release adoption will be messed up.
not sure if we should mention it though